dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.webBundleSecurity-1.0.feature

```diff
@@ -0,0 +1,16 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+
+symbolicName = io.openliberty.webBundleSecurity-1.0
```

Comment:
I  suggest we call this io.openliberty.webBundleSecurity.javaee-1.0
That distinguishes it more from the main webBundleSecurity feature.